var django_2contrib_2gis_2geos_2error_8py =
[
    [ "django.contrib.gis.geos.error.GEOSException", "classdjango_1_1contrib_1_1gis_1_1geos_1_1error_1_1_g_e_o_s_exception.html", null ]
];